modules/social_features/social_event/modules/social_event_managers/src/Element/SocialEnrollmentAutocomplete.php

```diff
@@ -37,13 +37,14 @@ public static function validateEntityAutocomplete(array &$element, FormStateInte
       $nid = $node;
     }
 
-    // Grab all the input values so we can get the ID's out of them.
-    $input_values = Tags::explode($element['#value']);
-
     // If we use the select 2 widget then we already got a nice array.
     if ($select2 === TRUE) {
```

Comment:
In general we don't really like if else statements and rather work without.. 

Would it potentially be better to throw the if statement around?

e.g.

`
$input_values = $element['#value'];
if ($select2 !== TRUE) {
  $input_values = Tags::explode($element['#value']);
}
`

Reply:
@ronaldtebrake PR updated.